skimage/feature/peak.py

```diff
@@ -187,18 +178,15 @@ def peak_local_max(image, min_distance=1, threshold_abs=None,
     Returns
     -------
     output : ndarray or ndarray of bools
-
-        * If `indices = True`  : (row, column, ...) coordinates of peaks.
-        * If `indices = False` : Boolean array shaped like `image`, with peaks
-          represented by True values.
+        The coordinates of the peaks.
 
     Notes
     -----
     The peak local maximum function returns the coordinates of local peaks
-    (maxima) in an image. Internally, a maximum filter is used for finding local
-    maxima. This operation dilates the original image. After comparison of the
-    dilated and original image, this function returns the coordinates or a mask
-    of the peaks where the dilated image equals the original image.
+    (maxima) in an image. Internally, a maximum filter is used for finding
+    local maxima. This operation dilates the original image. After comparison
+    of the dilated and original image, this function returns the coordinates or
+    a mask of the peaks where the dilated image equals the original image.
```

Comment:
Oups, I forgot to validate my suggestion :sweat_smile: 
```suggestion
    of the dilated and original image, this function returns the coordinates
    of the peaks where the dilated image equals the original image.
```